packages/react-components/src/templates/__tests__/DashboardPageBody.test.tsx

```diff
@@ -20,42 +18,13 @@ const props: ComponentProps<typeof DashboardPageBody> = {
       type: 'Event',
     },
   ],
-  pages: [createPageResponse('1'), createPageResponse('2')],
   userId: '42',
   teamId: '1337',
   roles: [],
   reminders: [],
   dismissedGettingStarted: false,
 };
 
-it('renders multiple news cards', () => {
-  render(<DashboardPageBody {...props} />);
```

Comment:
This test's props are crappy. We are not injecting the news we assert on within the specific test they are asserted on. Please inject news articles with specific titles you assert on here